public/video-ui/src/components/Videos/VideoItem.js

```diff
@@ -40,29 +41,56 @@ export default class VideoItem extends React.Component {
 
   render() {
     const video = this.props.video;
-    const scheduledLaunch = video.contentChangeDetails.scheduledLaunch && video.contentChangeDetails.scheduledLaunch.date;
+    const scheduledLaunch =
+      video.contentChangeDetails.scheduledLaunch &&
+      video.contentChangeDetails.scheduledLaunch.date;
+    const embargo =
+      video.contentChangeDetails.embargo &&
```

Comment:
could we extract this logic to the `VideoUtils` helper? https://github.com/guardian/media-atom-maker/blob/master/public/video-ui/src/util/video.js